playgrounds/angular/tsconfig.spec.json

```diff
@@ -0,0 +1,18 @@
+/* To learn more about this file see: https://angular.io/config/tsconfig. */
```

Comment:
Pure JSON does not support comments. It's better to remove it to be sure that everything works ok